### DISCUSSION TITLE
Increasing the amount of experience required.

### DISCUSSION THREAD

**Discussion by Prostocvasha** [FakeFishGames/Barotrauma#9181] [Ideas and suggestions]
I think the current cost of raising the level is too low, and talents for additional experience are useless.

My last two campaigns have shown that most players already have at least two-thirds of the talents by the middle of the map. And this is for a direct route to the end. Talents are too easy to get, and their conditional value is quite low. For 2-5 people, this is quite good, but with a larger team, I want to see more highly specialized players who will complement each other.

I propose to increase the basic required experience to increase the level by x2-x3 and add a slider modifier of this value to the settings when creating a campaign.

**Comment by ViktorShield** [comment]
I suggest making an exp gain modifier slider instead

**Reply by Prostocvasha** [reply]
This will affect talents for additional experience. I don't think it's possible.
And the required amount of experience to move to the next level is only a figure, right?
